configs/mainnet.yaml

```diff
@@ -188,3 +191,4 @@ MAX_REQUEST_PAYLOADS: 128
 ATTESTATION_DEADLINE: 4
 PROPOSER_INCLUSION_LIST_CUT_OFF: 11
 VIEW_FREEZE_DEADLINE: 9
+MAX_BLOBS_PER_BLOCK_EIP7805: 9
```

Comment:
I do not believe we should define new blob limits for FOCIL.

Reply:
Hello @jtraglia, thanks for your explanation! Now both PRs got simpler, which is really cool. I believe all we want is to have eip7805 fork so we can test it independently from other canonical forks, so please let me know if there is any more thing to subtract :)